### PR TITLE
[Concept Entry] Sklearn multiclass-classification

### PR DESCRIPTION
**Pull request by SaviDahegaonkar**
### Description

Created a new concept entry on the Multiclass Classification concept under Sklearn.

### Issue Solved
#4966 

### Type of Change

- Adding a new entry

### Checklist

- [ ] All writings are my own.
- [ ] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [ ] I have performed a self-review of my own writing and code.
- [ ] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/sklearn/concepts/multiclass-classification/multiclass-classification.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+model.fit(X_train, y_train)
+
+# Make predictions on the test data
+y_pred = model.predict(X_test)
+
+# Evaluate the model by calculating accuracy
+accuracy = accuracy_score(y_test, y_pred)
+
+# Print the accuracy of the model
+print(f"Accuracy: {accuracy:.2f}")
```
Comment (mamtawardhani) [inline content/sklearn/concepts/multiclass-classification/multiclass-classification.md:125]:
```suggestion
from sklearn.ensemble import RandomForestClassifier
from sklearn.datasets import make_classification
from sklearn.model_selection import train_test_split
from sklearn.metrics import accuracy_score

# Generate synthetic data for multiclass classification (3 classes)
X, y = make_classification(n_samples=1000, n_features=20, n_informative=4, n_classes=3, random_state=42)

# Split the dataset into training and testing sets (80% train, 20% test)
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)

# Initialize the RandomForestClassifier
model = RandomForestClassifier()

# Train the model on the training data
model.fit(X_train, y_train)

# Make predictions on the test data
y_pred = model.predict(X_test)

# Evaluate the model by calculating accuracy
accuracy = accuracy_score(y_test, y_pred)

# Print the accuracy of the model
print(f"Accuracy: {accuracy:.2f}")
```

**Comment by SaviDahegaonkar** [comment]
Hey @mamtawardhani ,
Incorporated the changes suggested by you.

Thanks,
Savi

File: content/sklearn/concepts/multiclass-classification/multiclass-classification.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
```
Comment (mamtawardhani) [inline content/sklearn/concepts/multiclass-classification/multiclass-classification.md:91]: You can simply add the syntax of RandomForestClassifier:

please refer to this: https://scikit-learn.org/1.5/modules/generated/sklearn.ensemble.RandomForestClassifier.html#sklearn.ensemble.RandomForestClassifier

Why do we need the iris dataset in syntax? @SaviDahegaonkar 
This should be a generic syntax

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Please change the syntax

**Comment by SaviDahegaonkar** [comment]
Will do it now.

**Review by mamtawardhani — APPROVED** [review]
Thank you for contributing to Codecademy Docs @SaviDahegaonkar  😄 

The entry looks good to be merged! 🚀

**Pull request merged**